Samples/V1.0Samples/LocalMediaSamples/PolicyRecordingBot/FirewallREADME.md

```diff
@@ -0,0 +1,47 @@
+Step 1: Create a Virtual Network and Subnet for Azure Firewall**
```

Comment:
Minor: Steps should be the heading and the next line the sub heading. Instead of bolding, 
lets use # (heading 1) and ## (heading2) respectively.